packages/test/test-driver-definitions/src/index.ts

```diff
@@ -0,0 +1,9 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export * from "./interfaces";
+export * from "./localServerTestDriver";
+// eslint-disable-next-line import/export
```

Comment:
What does this do?  `import/export` isn't the name of a rule that I can find.

Reply:
not sure. removing it.